doc-files/release-notes-24.md

```diff
@@ -0,0 +1,221 @@
+[JDK-8288893](https://bugs.openjdk.org/browse/JDK-8288893) | Popup and its subclasses cannot input text from InputMethod | scenegraph
+[JDK-8335470](https://bugs.openjdk.org/browse/JDK-8335470) | [XWayland] JavaFX tests that use AWT Robot fail on Wayland | swing
+[JDK-8340005](https://bugs.openjdk.org/browse/JDK-8340005) | Eliminate native access calls from javafx.swing | swing
+[JDK-8340849](https://bugs.openjdk.org/browse/JDK-8340849) | [macos] Crash when creating a child window of a JavaFX window after Platform::exit | swing
+[JDK-8328994](https://bugs.openjdk.org/browse/JDK-8328994) | Update WebKit to 619.1 | web
+[JDK-8334124](https://bugs.openjdk.org/browse/JDK-8334124) | Rendering issues with CSS "text-shadow" in WebView | web
+[JDK-8336941](https://bugs.openjdk.org/browse/JDK-8336941) | Update libxslt to 1.1.42 | web
+[JDK-8337481](https://bugs.openjdk.org/browse/JDK-8337481) | File API: file.name contains path instead of name | web
+[JDK-8338307](https://bugs.openjdk.org/browse/JDK-8338307) | Additional WebKit 619.1 fixes from WebKitGTK 2.44.3 | web
+[JDK-8340208](https://bugs.openjdk.org/browse/JDK-8340208) | Additional WebKit 619.1 fixes from WebKitGTK 2.44.4 | web
+[JDK-8087863](https://bugs.openjdk.org/browse/JDK-8087863) | Mac: "Select All" within ListView/TreeView is handled differently depending on the useSystemMenuBar value | window-toolkit
+[JDK-8273743](https://bugs.openjdk.org/browse/JDK-8273743) | KeyCharacterCombination for "+" does not work on US QWERTY keyboard layout | window-toolkit
+[JDK-8319779](https://bugs.openjdk.org/browse/JDK-8319779) | SystemMenu: memory leak due to listener never being removed | window-toolkit
+[JDK-8325445](https://bugs.openjdk.org/browse/JDK-8325445) | [macOS] Colors are not displayed in sRGB color space | window-toolkit
+[JDK-8332222](https://bugs.openjdk.org/browse/JDK-8332222) | Linux Debian: Maximized stage shrinks when opening another stage | window-toolkit
+[JDK-8333919](https://bugs.openjdk.org/browse/JDK-8333919) | [macOS] dragViewOffsetX/dragViewOffsetY are ignored for the dragView image | window-toolkit
+[JDK-8335469](https://bugs.openjdk.org/browse/JDK-8335469) | [XWayland] crash when an AWT ScreenCast session overlaps with an FX ScreenCast session | window-toolkit
+[JDK-8339178](https://bugs.openjdk.org/browse/JDK-8339178) | [macos] Swing InterOp Platform.exit() crash | window-toolkit
+[JDK-8339183](https://bugs.openjdk.org/browse/JDK-8339183) | [macos] Premature exit in Swing interop when last JFrame is disposed | window-toolkit
+[JDK-8340982](https://bugs.openjdk.org/browse/JDK-8340982) | [win] Dead key followed by Space generates two characters instead of one | window-toolkit
+[JDK-8344372](https://bugs.openjdk.org/browse/JDK-8344372) | Setting width for TRANSPARENT Stage -> gtk_window_resize: assertion 'height > 0' | window-toolkit
+[JDK-8348744](https://bugs.openjdk.org/browse/JDK-8348744) | Application window not always activated on macOS 15 | window-toolkit
```

Comment:
double checked all the links.

BTW, are these links (`[JDK-8348744](https://bugs.openjdk.org/browse/JDK-8348744)`) formatted manually or is there some kind of script?

Reply:
as long as it's not a totally manual process, we should be fine.  otherwise we'd need to check the links every time.